cmd/operator-sdk/migrate/cmd.go

```diff
@@ -23,20 +23,28 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/ansible"
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/helm"
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/project"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+	"github.com/pkg/errors"
```

Comment:
Did we mean to use `"github.com/pkg/errors"` or should this be just `"errors"`

Reply:
Yes for its `Wrap(f)` functions.